mava/utils/make_env.py

```diff
@@ -212,6 +228,44 @@ def make_gigastep_env(
     return train_env, eval_env
 
 
+def make_gym_env(
+    config: DictConfig,
+    num_env: int,
+    add_global_state: bool = False,
+) -> GymToJumanji:
+    """
+     Create a gymnasium environment.
+
+    Args:
+        config (Dict): The configuration of the environment.
+        num_env (int) : The number of parallel envs to create.
+        add_global_state (bool): Whether to add the global state to the observation. Default False.
+
+    Returns:
+        Async environments.
+    """
+    wrapper = _gym_registry[config.env.env_name]
+    config.system.add_agent_id = config.system.add_agent_id & (~config.env.implicit_agent_id)
+
+    def create_gym_env(config: DictConfig, add_global_state: bool = False) -> gymnasium.Env:
+        registered_name = f"{config.env.scenario.name}:{config.env.scenario.task_name}"
+        env = gym.make(registered_name, disable_env_checker=False, **config.env.kwargs)
```

Comment:
Don't think we need this checker, especially since it's not jitted I'd like to save as much time as possible
```suggestion
        env = gym.make(registered_name, disable_env_checker=True, **config.env.kwargs)
```